### DISCUSSION TITLE
Zola

### DISCUSSION THREAD

**Discussion by uncomfyhalomacro** [catppuccin/catppuccin#1923] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the requested port be themed?

By following https://www.getzola.org/documentation/themes/overview/

### Has the requested port been themed already?

It's a **work in progress** in https://github.com/uncomfyhalomacro/catppuccin-zola. It may not be done in a week. Busy month for me. But any helping hand can help :smile:

### Any additional comments?

I've done a bit zola theming and exploring around so if anyone has any questions, feel free to ask. I am no front-end professional though so I hope someone can give feedback or PR on how to improve the theme.

**Comment by justTOBBI** [comment]
If you want, I can help with some stuff.
I would propose a new README soon

**Reply by uncomfyhalomacro** [reply]
Sure! 👀
